lib/new_relic/agent/threading/agent_thread.rb

```diff
@@ -10,6 +10,7 @@ class AgentThread
         def self.create(label, &blk)
           ::NewRelic::Agent.logger.debug("Creating New Relic thread: #{label}")
           wrapped_blk = Proc.new do
+            ::NewRelic::Agent.logger.warn("AgentThread created with current transaction #{::Thread.current[:newrelic_tracer_state].current_transaction.best_name}") if ::Thread.current[:newrelic_tracer_state] && ::Thread.current[:newrelic_tracer_state].current_transaction
```

Comment:
Why did you decide to print to the `warn` level log here?

Reply:
I like that you called this AgentThread since that's the name of the class. Even though it's not part of your changes, if you're up for it, I'd recommend changing the references in the other log messages from `New Relic thread` to `AgentThread`.

Reply:
I did warn level because if it does happen it's not immediately breaking anything, which is what I would use error for. But it can still cause issues in some applications, so seemed more important than info or debug level. 

And sure, I can make that change, I think that makes sense!

Reply:
resolved in 32ca4edde262e868cc7098c2774cb691d69826b9

Reply:
Do we not have a line length linter check? I think for now it'd be nice to stick to at least an upper boundary of 120 columns.

Reply:
updated in 011b14e6ec9f9bc4567aa3d849866c7fc876021e